Cargo.lock

```diff
@@ -122,6 +122,12 @@ version = "0.2.16"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0942ffc6dcaadf03badf6e6a2d0228460359d5e34b57ccdc720b7382dfbd5ec5"
 
+[[package]]
+name = "android-tzdata"
```

Comment:
I suppose so, it seems to be a dep of `chrono`, which Poem depends on. It might be possible to disable needing that dep (it is marked as optional in the Cargo.toml for Poem) but can we leave that as a followup for later?